modules/cats/src/smithy4s/interopcats/SchemaVisitorShow.scala

```diff
@@ -0,0 +1,150 @@
+package smithy4s.interopcats
+
+import cats.Show
+import cats.implicits.toContravariantOps
+import smithy4s._
+import smithy4s.capability.EncoderK
+import smithy4s.interopcats.instances.ShowInstances._
+import smithy4s.schema.{
+  Alt,
+  CollectionTag,
+  EnumValue,
+  Field,
+  Primitive,
+  Schema,
+  SchemaAlt,
+  SchemaField,
+  SchemaVisitor
+}
+import smithy4s.schema.Alt.Precompiler
+
+object SchemaVisitorShow extends SchemaVisitor[Show] { self =>
+
+  override def primitive[P](
+      shapeId: ShapeId,
+      hints: Hints,
+      tag: Primitive[P]
+  ): Show[P] = primShowPf(tag)
+
+  override def collection[C[`2`], A](
```

Comment:
huh, looks like it actually compiles. Where is this from though ? 

Even though it works, please change to `def collection[C[_]]`